src/modules/dashboard/staff/staff-view.vue

```diff
@@ -31,37 +40,61 @@ import {
 } from '@/global/composables/useDialog';
 import { DialogTypeEnum } from '@/global/enums/dialog-type.enum';
 import { Message } from '@/global/models/message';
+import { staffService } from './service/staff.service';
+import { ActionType } from '../enums/action-type.enum';
 import StaffPopup from './components/staff-popup.vue';
 import Loader from '../components/loader.vue';
 import Popup from '@/global/components/popup.vue';
 import i18n from '@/plugins/i18n';
+import ActionBar from './components/action-bar.vue';
+import Fuse from 'fuse.js';
 
 const globalStore = useGlobalStore();
 const staffStore = useStaffStore();
-const isStaffEditVisible = ref(false);
+const isStaffFormVisible = ref(false);
+const selectedStaffId = ref(0);
+const popupType = ref(ActionType.ADD);
+const searchQuery = ref('');
 
-const { isLoading } = useQuery({
+const { isLoading, refetch } = useQuery({
   queryKey: ['getStaffMembers'],
```

Comment:
Nie zastanawiałeś się nad tym aby utrzymywać klucze gdzieś globalnie? Może przyjdzie potrzeba w innym miejscu aplikacji zrobić refetch staffu po jakiejs innej akcji, wtedy znowu z palca ustawisz klucz do odświeżenia, a tak to będziesz miał pod constem tego stringa.